rainforest/environments.go

```diff
@@ -16,9 +18,17 @@ type Environment struct {
 
 // CreateTemporaryEnvironment creates a new temporary environment and returns the
 // Environment.
-func (c *Client) CreateTemporaryEnvironment(urlString string) (*Environment, error) {
+func (c *Client) CreateTemporaryEnvironment(runDescription string, urlString string) (*Environment, error) {
+	name := "temporary-env-for-custom-url-via-CLI"
+	if runDescription != "" {
+		if len(runDescription) > 241 {
```

Comment:
A bit late to the party, but probably worth using a constant instead of `241` to make it clearer what's going on.